modules/gltf/SCsub

```diff
@@ -3,6 +3,23 @@
 Import("env")
 Import("env_modules")
 
+# For MP3 support in the audio GLTFDocumentExtension.
+if env["module_minimp3_enabled"]:
+    thirdparty_dir = "#thirdparty/minimp3/"
+    if not env.msvc:
+        env_modules.Append(CPPFLAGS=["-isystem", Dir(thirdparty_dir).path])
+    else:
+        env_modules.Prepend(CPPPATH=[thirdparty_dir])
+    if not env["minimp3_extra_formats"]:
+        env_modules.Append(CPPDEFINES=["MINIMP3_ONLY_MP3"])
+
+# For OGG Vorbis support in the audio GLTFDocumentExtension.
+if env["module_vorbis_enabled"]:
+    thirdparty_dir = "#thirdparty/libvorbis/"
+    env_modules.Prepend(CPPPATH=[thirdparty_dir])
+    if env["builtin_libogg"]:
+        env_modules.Prepend(CPPPATH=["#thirdparty/libogg"])
```

Comment:
I could use some assistance from buildsystem gurus here. Right now this code adds these path includes into the whole of the modules environment, which seems wrong. If I move `env_gltf = env_modules.Clone()` above these lines, and change `env_modules` here to `env_gltf`, it does not compile.